composables/autoTeleport/useAutoTeleportTransitionDetails.ts

```diff
@@ -135,7 +135,7 @@ export default function (
       !teleportTxFee.value &&
       addTeleportFee.value &&
       hasEnoughInRichestChain.value &&
-      amountToTeleport.value,
+      amountToTeleport.value >= 0,
```

Comment:
**question**: why `= 0` works here?

Reply:
it doesn't , it was already checking = 0 before, mb